images/runtime/python/template.Dockerfile

```diff
@@ -95,7 +106,7 @@ RUN pip install --upgrade pip \
     && pip install viztracer==0.15.6 \
     && pip install vizplugins==0.1.3 \
     # Removing orjson only for 3.12 due to build errors
-    && if [ "${PYTHON_VERSION}" != "3.12" ] && [ "${PYTHON_VERSION}" != "3.7" ]; then pip install orjson==3.10.1; fi \
+    && if [ "${PYTHON_VERSION}" != "3.12" ] && [ "${PYTHON_VERSION}" != "3.7" ]; then pip install orjson==3.10.7; fi \
```

Comment:
any specific reason for updating orjson here?

Reply:
while generating python 3.13 image, it is giving build error with older version of orjson.